components/match2/wikis/valorant/legacy/match_maps_legacy.lua

```diff
@@ -76,24 +77,19 @@ function MatchMapsLegacy._handleMaps(args)
 			return false
 		end
 
-		local score1
-		local score2
 		if Logic.isNotEmpty(score) then
-			local splitedScore = mw.text.split(score, '-')
-			score1 = splitedScore[1]
-			score2 = splitedScore[2]
-		end
-		if not score1 and not score2 and winner ~= SKIP then
-			local winnerInt = tonumber(winner)
-			score1 = winnerInt == 1 and DEFAULT_WIN or DEFAULT_LOSS
-			score2 = winnerInt == 2 and DEFAULT_WIN or DEFAULT_LOSS
+			local splitedScore = Array.parseCommaSeparatedString(score, '-')
+			args[prefix .. 'score1'] = splitedScore[1] or ''
+			args[prefix .. 'score2'] = splitedScore[2] or ''
```

Comment:
is `or ''` needed?

Reply:
In cs "map1score=W-" was a common thing.

Reply:
Will anything break without it?

Reply:
not really